activesupport/lib/active_support/duration/iso8601_parser.rb

```diff
@@ -9,8 +9,6 @@ class Duration
     #
     # This parser allows negative parts to be present in pattern.
     class ISO8601Parser # :nodoc:
-      class ParsingError < ::ArgumentError; end
```

Comment:
Do we think people are catching this exception today? Do we need to add back an alias (with a deprecation)? Or is this minor enough that we could just break in 5.1?